.github/workflows/CI-unittests.yml

```diff
@@ -21,30 +21,6 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: ubuntu-latest
```

Comment:
Do you really want to stop testing / remove support for all versions below 3.11?

I'd say it's your prerogative to do so, but we usually support more than just a single python version and test against all supported ones.

NEP 29 provides a schedule for dropping python version support for packages, which we e.g. follow in ctapipe and there the current recommendation is to support >=3.9.

If you have strong reasons to require a new one, that's fine as well.